.github/workflows/deployment.yaml

```diff
@@ -215,6 +215,7 @@ jobs:
           export STORAGE_ACCOUNT=phdi${TF_ENV}phi${SHORT_CID}
           export KEY_VAULT=${TF_ENV}vault${SHORT_CID}
           export MPI_DB_HOST=phdi${TF_ENV}mpi${SHORT_CID}.postgres.database.azure.com
+          export KEY_VAULT_LINKED_SERVICE=${terraform.workspace}${substr(var.client_id, 0, 8)}-keyvault-linked-service
```

Comment:
Do we need this env var? I don't see changes introduced to the notebooks that would take advantage of this. I could definitely be missing something here though.

Reply:
Good q! I updated this branch with changes from the pipeline automation so hopefully it is clearer where this var is being used.